docs/opcodes/31/berlin.mdx

```diff
@@ -0,0 +1,3 @@
+## Gas
+
+The static cost is {gasPrices|balance}. If the accessed address is warm, the dynamic cost is {gasPrices|warmstorageread}. Otherwise the dynamic cost is {gasPrices|coldaccountaccess}. See section [access sets](/about).
```

Comment:
Very nit: Maybe we could add the formular here like gas = n cold accesses * {gasPrices|coldaccountaccess} + n warm accesses * {gasPrices|warmstorageread}